batch/test/test_accounts.py

```diff
@@ -209,7 +209,7 @@ async def test_close_billing_project_with_open_batch_errors(
     project = new_billing_project
     await dev_client.add_user("test", project)
     client = await make_client(project)
-    b = await client.create_batch()._create()
+    b = await client.create_batch()._open_batch()
```

Comment:
I'm concerned that we've taken away the tests for the code where we use the slower code path. Can we add a test that makes the max_bunch_size 1 and submit a batch with 2 or 3 jobs?